src/interaction.jl

```diff
@@ -82,17 +80,46 @@ function bubbledyson(V::Float64, F::Float64, Π::Float64, n::Int)
     return K
 end
 
+function bubbledysonreg(V::Float64, F::Float64, Π::Float64, n::Int; regV::Float64 = V)
+    # V:bare interaction
+    # F:F^{+-} is local field factor,0 for RPA
+    # Π:Polarization. 2*Polarization0 for spin 1/2
+    # n:matfreq. special case for n=0
+    # comparing to previous convention, an additional V is multiplied
+    K = 0
+    if V ≈ 0
+        K = Π * ( - F)^2 / (1.0 - (Π) * ( - F)) / regV
+        return K
+    end
+    if n == 0
+        if F == 0
+            K = Π * (1)^2 / (1.0 / V - Π * (1))
+        else
+            K = Π * (1 - F / V)^2 / (1.0 / V - Π * (1 - F / V))
+        end
+    else
+        K = Π * (V - F)^2 / (1.0 - (Π) * (V - F)) / regV
```

Comment:
do not do something like V/reqV, because there will be a digit loss already at the step of V